src/applications/coronavirus-vaccination-expansion/config/PreSubmitInfo.js

```diff
@@ -58,20 +34,46 @@ const PreSubmitCheckboxes = ({ showError, onSectionComplete }) => {
 
   return (
     <>
-      <Checkbox
+      <VaCheckbox
+        id="privacy-practices-checkbox"
         checked={privacyAgreementAccepted}
-        onValueChange={value => setPrivacyAgreementAccepted(value)}
-        label={privacyLabel}
-        errorMessage={privacyError && privacyErrorMessage}
+        onVaChange={event => setPrivacyAgreementAccepted(event.detail.checked)}
+        label="I have been provided access to VA's Notice of Privacy Practices."
+        error={privacyErrorMessage}
         required
-      />
-      <Checkbox
+      >
+        <span slot="description">
+          <a
+            id="kif-privacy-policy"
+            target="_blank"
+            rel="noreferrer"
+            href="https://www.va.gov/files/2022-10/10-163p_(004)_-Notices_of_Privacy_Practices-_PRINT_ONLY.pdf"
```

Comment:
We have updated this link because the previous one does not work:

https://www.va.gov/vhapublications/ViewPublication.asp?pub_ID=1090